### ISSUE TITLE
Nexus homedir path is hard coded

### ISSUE THREAD

**Issue by nicosto** [ansible-ThoTeam/nexus3-oss#129]
Hi,

In the file nexus_install.yml
Nexus home directory is hard coded and break if nexus user is not under /home

```
- name: Set NEXUS_HOME for the service user
  lineinfile:
    dest: "/home/{{ nexus_os_user }}/.bashrc"
    regexp: "^export NEXUS_HOME=.*"
    line: "export NEXUS_HOME={{ nexus_installation_dir }}/nexus-latest"
  notify:
    - nexus-service-stop
```

**Comment by zeitounator** [comment]
Thanks for this feedbask @nicosto. Definitely a must fix. Will try to release soon.

**Comment by zeitounator** [comment]
I must be tired, just noticed your proposed a PR. Going to check there.

**Comment by zeitounator** [comment]
Fixed in #130
